core/src/account_provider.rs

```diff
@@ -264,7 +264,7 @@ mod tests {
     fn unlock_account_temp() {
         let kp = Random.generate().unwrap();
         let ap = AccountProvider::transient_provider();
-        assert!(ap.insert_account(*kp.private(), &"test".into()).is_ok());
```

Comment:
Why do you prohibit copy?

Reply:
It is a very low-level security concern and `Rust` can safely dispose private information by implementing memory clear logic in `Drop` trait. However, the more conservative way is prohibiting copy behavior of private information. Previously, private information was stored in `H256` type and the `DecryptedAccount` which possibly contains private key info implements memory-clearing `Drop` trait.
However, I could not sure about other usages are desirable because `H256` was used just for hash information also. And this change did not require many changes. What do you think of it?

Reply:
@HoOngEe How about adding summary of this description in the commit message?

Reply:
I'm not sure the purpose yet. Prohibiting a copy of private data is not a way to keep data privately. I can agree that H256 seems to general and you want to replace it with `SecretKey`, but it doesn't mean that SecretKey is safer because it doesn't provide `Copy`.

Reply:
Oh my god. I misunderstood the types. The actual situation is: `SecretKey` by `sodiumoxide` implemented memory clearing `Drop` trait inside so the `Copy` marker is incompatible. Therefore, the `Private` which is a wrapper for `SecretKey` cannot mark `Copy` on it. In the sense, this change is inevitable. Is there some use cases need to allow `Copy` behavior?

Reply:
No, we don't need `Copy`, we can use `Clone` instead of it.

Reply:
Please change the commit message.

Reply:
I modified the commit message to be correct.